syncmaster/scheduler/transfer_job_manager.py

```diff
@@ -26,7 +26,7 @@ def update_jobs(self, transfers: list[Transfer]) -> None:
             job_id = str(transfer.id)
             existing_job = self.scheduler.get_job(job_id)
 
-            if not transfer.is_scheduled or transfer.is_deleted:
+            if not transfer.is_scheduled:
```

Comment:
In this case scheduler will not remove scheduled job. Is that okay?

Reply:
didn't catch, why scheduler should remove scheduled jobs?

Reply:
If scheduled transfer is deleted from the table using UI/API, and after that scheduler will scan `transfers` table to update scheduler jobs list, it will not see row that were deleted.